docs/website/docs/hub/intro.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Introduction
```

Comment:
we should probably have a redirect from `docs/devel/hub/` to `docs/devel/hub/intro`